### ISSUE TITLE
Can the "base 16" colors be changed at runtime through Ansi escape sequences?

### ISSUE THREAD

**Issue by cwegener** [raphamorim/rio#188]
I use https://github.com/chriskempson/base16-shell as my color scheme management.

The concept behind base16-shell is that it will emit the necessary ANSI escape characters on shell launch to set the base 16 colors. base16-shell differentiates between different environments (TMUX, screen, linux console and "everything else") and will emit the corresponding ANSI escape characters according to the detected environment.

Can rio change its base 16 colors in response to ANSI escape characters similar to other terminal emulators?

**Comment by raphamorim** [comment]
No, it doesn't have support for it today but would be a good to have. Will mark it as wishlist.

**Comment by cwegener** [comment]
Just updating with a link to the successor project to the original base16 project:

https://github.com/tinted-theming/base16-shell

**Comment by raphamorim** [comment]
Update on this issue, @cwegener the ANSI instructions should been working now (I forgot about this issue, probably was fixed once implemented the missing instructions for PTYWrite). 

The thing is that Rio is completely ignoring color scheme overwrite from ANSI sequences https://github.com/raphamorim/rio/blob/main/rio/src/sequencer.rs#L216-L223

**Comment by raphamorim** [comment]
Ok time to prioritize it as well, moving to TODO

**Comment by raphamorim** [comment]
Could you folks test with latest main?

**Comment by cwegener** [comment]
Testing with `base16-shell`  https://github.com/chriskempson/base16-shell on Archlinux with a fresh `aur sync rio-git` works well! Thanks for that.

I don't have a macOS machine available for testing at the moment ..

**Comment by raphamorim** [comment]
Thank you @cwegener closing this issue, sorry for taking so long
